README.md

```diff
@@ -15,6 +15,16 @@ Docs: [![Documentation Status](https://inch-ci.org/github/samvera/sufia.svg?bran
 Jump in: [![Slack Status](http://slack.samvera.org/badge.svg)](http://slack.samvera.org/)
 [![Ready Tickets](https://badge.waffle.io/samvera/sufia.png?label=ready&title=Ready)](https://waffle.io/samvera/sufia)
 
+# End of Life Notice
```

Comment:
I wonder if we should replace pretty much everything in this README with this section so folks don't miss it.